xbmc/addons/AddonInstaller.h

```diff
@@ -135,6 +146,8 @@ class CAddonInstaller : public IJobCallback
   mutable CCriticalSection m_critSection;
   JobMap m_downloadJobs;
   CEvent m_idle;
+  // controls whether Kodi is already updating addons
+  std::atomic<bool> m_updatingAddons{false};
```

Comment:
Seen my runtime test, that the atomic makes this and need above the `AddonInstaller.h` a `#include <atomic>` by me on Linux.:
```
In file included from /home/alwin/Development/Kodi/kodi-Matrix/xbmc/games/dialogs/GUIDialogSelectGameClient.cpp:12:
/home/alwin/Development/Kodi/kodi-Matrix/xbmc/addons/AddonInstaller.h:150:21: error: field ‘m_updatingAddons’ has incomplete type ‘std::atomic<bool>’
  150 |   std::atomic<bool> m_updatingAddons{false};
      |                     ^~~~~~~~~~~~~~~~
In file included from /usr/include/c++/9/bits/shared_ptr_atomic.h:33,
                 from /usr/include/c++/9/memory:82,
                 from /home/alwin/Development/Kodi/kodi-Matrix/xbmc/games/GameTypes.h:11,
                 from /home/alwin/Development/Kodi/kodi-Matrix/xbmc/games/dialogs/GUIDialogSelectGameClient.h:11,
                 from /home/alwin/Development/Kodi/kodi-Matrix/xbmc/games/dialogs/GUIDialogSelectGameClient.cpp:9:
/usr/include/c++/9/bits/atomic_base.h:145:12: note: declaration of ‘struct std::atomic<bool>’
  145 |     struct atomic;
      |            ^~~~~~
```